assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -98,6 +147,16 @@
   }
 }
 
+@media only all and (max-width: 730px) {
```

Comment:
The mobile menu should be visible until 899px, the desktop view should start after 900px screen sizes. Curently the mobile menu is until 729px and you can see the desktop view after 730px

Reply:
I made the menu visible from 899px, and the desktop view from 900px and I used the function suggested by you